src/elements/a-gltf-entity.js

```diff
@@ -102,6 +102,19 @@ const inflateEntities = function(classPrefix, parentEl, node) {
 
   el.setObject3D(node.type.toLowerCase(), node);
 
+  // Set the name of the `THREE.Group` to match the name of the node,
+  // so that `THREE.PropertyBinding` will find (and later animate)
+  // the group. See `PropertyBinding.findNode`:
+  // https://github.com/mrdoob/three.js/blob/dev/src/animation/PropertyBinding.js#L211
+  el.object3D.name = node.name;
+  if (node.animations) {
+    // Pass animations up to the group object so that when we can pass the group as
+    // the optional root in `THREE.AnimationMixer.clipAction` and use the hierarchy
+    // preserved under the group (but not the node). Otherwise `clipArray` will be
+    // `null` in `THREE.AnimationClip.findByName`.
+    node.parent.animations = node.animations;
```

Comment:
I still think this hack is crazy. Thanks, for documenting it. Hopefully we can find a more robust way of doing this in the future. This could screw up [object3D.getObjectByName](https://threejs.org/docs/index.html#api/core/Object3D.getObjectByName) calls.